### ISSUE TITLE
Rollout Advanced Group Definition shows different Groups in Rollout overview.

### ISSUE THREAD

**Issue by ghost** [eclipse-hawkbit/hawkbit#1400]
Hi all,

I am pretty sure I found a bug. 

## Reproduction
Construct a Rollout in the Management UI using *Advanced Group definition*. This yields a correct distribution of Targets for us:
![rollout_1_redact](https://github.com/eclipse/hawkbit/assets/100685413/ac7bae72-7c73-494a-8a1c-e04c720c49bf)

However, if I save the Rollout, the groups are shown with a different distribution:
![rollout_2_redact](https://github.com/eclipse/hawkbit/assets/100685413/61b7c97e-2b3e-4108-b10d-e0c6fc77b8a7)

## Expectation
The overview and the details view of the Rollout should show the same distribution.

We are using 0.3.0M7, as we did not get to migrate yet (the new *User Consent Flow* does not seem optional). If you need further information, please let me know.

**Comment by strailov** [comment]
Hey @dcd-arnold 
Thanks for using hawkBit! 

What do you mean by "the new User Consent Flow does not seem optional" ? Currently I think there is enable/disable option in system config, but once enabled could not be disabled again.

**Comment by dcd-arnold** [comment]
Hi @strailov,

thanks for *making* hawkbit.

I did not find the setting. The [Release Notes](https://github.com/eclipse/hawkbit/releases/tag/0.3.0M8) only mentions a setting by device. We are managing 700+ devices. Moreover, they are deleted and re-register themselves as needed. The [rauc-hawkbit-updater](https://github.com/rauc/rauc-hawkbit-updater) does not support to set the setting on registration. Unless there is a setting for the whole hawkbit instance, we are stuck until we figured out a way moving forward. If there is such a setting, where is it documented?

Thanks in advance,
Arnold

**Comment by strailov** [comment]
Yes, this seems like a good catch ... 
Somehow we've missed this. 
You can read at least some basic info [https://github.com/eclipse/hawkbit/pull/1293](here) . 

And more specifically :
"The consent flow can be activated in the system configuration. By default, the update process is executed as usual. "

**Comment by dcd-arnold** [comment]
Ah. I see. So if I don't activate the new user consent flow, it will not be activated on upgrade. I will keep that in mind and try an upgrade.

**Comment by strailov** [comment]
Yes ... And for the documentation we will try updating it as soon as possible.
